src/CalibrateCMP/KiDUtils.jl

```diff
@@ -43,7 +43,7 @@ function run_KiD_multiple_cases(u::Array{FT, 1}, u_names::Array{String, 1}, conf
         config["model"]["w1"] = case.w1
         config["model"]["p0"] = case.p0
         config["model"]["Nd"] = case.Nd
-        if "t_cal" in collect(keys(case))
+        if :t_cal in collect(keys(case))
             config["model"]["filter"] = make_filter_props(
                 config["model"]["filter"]["nz_unfiltered"],
                 case.t_cal;
```

Comment:
Here, I think we need to add 
```
        else
            config["model"]["filter"] = make_filter_props(
                config["model"]["filter"]["nz_unfiltered"],
                config["model"]["t_calib"];
                apply = config["model"]["filter"]["apply"],
                nz_per_filtered_cell = config["model"]["filter"]["nz_per_filtered_cell"],
                nt_per_filtered_cell = config["model"]["filter"]["nt_per_filtered_cell"],
            )  
        end
```
        
 If we check with an example in which the 1st case **has** the `t_cal` flag, while the second case **doesn't** have the `t_cal `flag, then for the case without the flag it keeps the last filter (the one created for the case with t_cal), not the original one.
 
 Let me know if you agree and if it makes sense to you!

Reply:
You're right about the bug. But I guess the else statement introduces a new bug. Now if you have a third case without t_calib it will use the filter of the second case because the default filter has been replaced when the second case was being handled.

Reply:
Doesn't it make a new filter for every case in this way (not considering the default one anymore)?

Reply:
I agree and it makes total sense. Thank you!

Reply:
I can check with different combinations of cases with and without t_cal, but apart from that I think it looks good!